Source/WebKit/GPUProcess/media/RemoteAudioTrackProxy.cpp

```diff
@@ -75,7 +75,8 @@ void RemoteAudioTrackProxy::configurationChanged()
     if (!m_connectionToWebProcess)
         return;
 
-    m_connectionToWebProcess->connection().send(Messages::MediaPlayerPrivateRemote::RemoteAudioTrackConfigurationChanged(m_identifier, configuration()), m_mediaPlayerIdentifier);
+    m_connectionToWebProcess->connection().send(Messages::MediaPlayerPrivateRemote::RemoteAudioTrackConfigurationChanged(m_id, configuration()), m_mediaPlayerIdentifier);
+    m_id = m_trackPrivate->id();
```

Comment:
This looks like it's in the wrong order. Shouldn't `m_id` be reset _before_ sending the message?

Reply:
No. Because we send the old id to the content process along the new one. It can the. Determine if the Id changed

Reply:
I'll use std::exchange to make the code clearer.